build_external/bin/clean-install.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+DISTRO="$1"
+VERSION="$2"
+BuildBase="$(cd "$(dirname "$0")" && cd .. && pwd)"
+
+# This is temporary - not all of the package-builder images from the helper-images repo
+# are available on Docker Hub. When everything falls under the "happy case"  below this
+# can be deleted in a future iteration. This is written in a weird way for portability,
+# can't rely on bash 4.0+ to allow case fall-through with ;&
```

Comment:
I assume you're referring to the `*_i386` variants not being on Docker hub. Those are, for the moment, a hack to allow for building of 32-bit x86 packages until `docker buildx` is more stable. Unless we specifically need handling for 32-bit distros here, we can probably safely ignore them for this.

Reply:
No - it was Arch Linux that is not available at all as a tag under the image.

Reply:
The ones in the whitelist are taken from https://hub.docker.com/r/netdata/package-builders/tags although I also checked manually what I could pull and it seemed to match this list.

Reply:
Ah, OK, that's not likely to change at any time in the future. We're not building AUR packages ourselves and it's not a high priority to take over maintainership there as the person who's maintaining that within the Arch community is honestly doing an awesome job.